$NetBSD: patch-ab,v 1.1.1.1 2002/09/02 16:43:03 agc Exp $

Generalise for all *BSD (well, anything with Torek stdio, really)

--- rxfn.c	2002/09/02 08:50:31	1.1
+++ rxfn.c	2002/09/02 08:51:26
@@ -41,7 +41,7 @@
 # ifdef linux
 #  define _CNT(fp) ((fp)->_IO_read_end - (fp)->_IO_read_ptr)
 # else
-#  ifdef __FreeBSD__
+#  if (defined(BSD) && BSD >= 199306)
 #   define _CNT(fp) ((fp)->_r)
 #  else
 #   define _CNT(fp) ((fp)->_cnt)
